pkgs/development/python-modules/tensorboard/default.nix

```diff
@@ -24,15 +23,15 @@
 
 buildPythonPackage rec {
   pname = "tensorboard";
-  version = "2.11.0";
+  version = "2.12.2";
```

Comment:
```suggestion
  version = "2.13.0";
```
Any reason not to?

Reply:
Because it was not released when I made the PR.
I just updated it !